R/fit_hetgp.R

```diff
@@ -1,17 +1,26 @@
 #' Title
 #'
-#' @param df
-#' @param reference_datetime
-#' @param horizon
+#' @param X 
+#' @param Z
 #'
 #' @return
 #' @export
 #'
 #' @examples
 #'
-fit_hetgp <- function(df, reference_datetime, horizon){
-  #add stuff
+fit_hetgp = function(X, Z){
```

Comment:
are X and Z standard names?  How do you know the starting reference_datetime and horizon for the forecast?